crates/ide-assists/src/handlers/term_search.rs

```diff
@@ -0,0 +1,230 @@
+//! Term search assist
+use hir::term_search::TermSearchCtx;
+use ide_db::assists::{AssistId, AssistKind, GroupLabel};
+
+use itertools::Itertools;
+use syntax::{ast, AstNode};
+
+use crate::assist_context::{AssistContext, Assists};
+
+pub(crate) fn term_search(acc: &mut Assists, ctx: &AssistContext<'_>) -> Option<()> {
+    let unexpanded = ctx.find_node_at_offset::<ast::MacroCall>()?;
+    let syntax = unexpanded.syntax();
+    let goal_range = syntax.text_range();
+
+    let excl = unexpanded.excl_token()?;
+    let macro_name_token = excl.prev_token()?;
+    let name = macro_name_token.text();
+    if name != "todo" {
```

Comment:
This should fetch the actual todo macro and check whether the two are the same via `FamousDefs` (you can just look at other usages of FamousDefs that should show you what to do (example https://github.com/veykril/rust-analyzer/blob/5860763f1808ef7ba2628bf46c08e77c96804925/crates/ide/src/inlay_hints.rs#L572)

Reply:
Also might be nice to make this work on `todo` and `unimplemented` alike

Reply:
I'm having hard time locating the `todo` macro from standard library.
`core:macros:todo` seems to be the path but in reality it only finds macros that have `macro` keyword which means no `todo` macro.
Something like this is found: https://doc.rust-lang.org/stable/src/core/macros/mod.rs.html#144
But not this: https://doc.rust-lang.org/stable/src/core/macros/mod.rs.html#871-878

Reply:
Oh right, those behave different in regards to their path. For the macro rules ones you'll just need the path from the crate root, so `core::todo` should resolve I believe (as the `macro_export` will re-export them from the crate root of core)

Reply:
Ty, works like charm!
Any ideas how to use it in tests tho as there is no std lib?